src/app/beer_garden/scheduler.py

```diff
@@ -367,7 +367,7 @@ def execute_job(self, job_id, reset_interval=False, **kwargs):
         job = db.query_unique(Job, id=job_id)
         self.add_job(
             run_job,
-            trigger=DateTrigger(datetime.now(), timezone="UTC"),
+            trigger=DateTrigger(datetime.utcnow(), timezone="UTC"),
```

Comment:
I assume this works as intended, but unless there's some compelling reason not to which should just remove the trigger (and presumably trigger_type) as the [official documentation](https://apscheduler.readthedocs.io/en/3.x/userguide.html#adding-jobs) suggests.  It specifically says:

>**Tip:**  To run a job immediately, omit trigger argument when adding the job.

Reply:
Alas, cannot change at this time. The `add_job` function specifically checks that the `trigger` argument is not `None`. Though I could change that function too, I'm not confident that such a change wouldn't break something else unexpectedly.